trunk/SUAVE/Analyses/Mission/Vary_Cruise/Given_State_of_Charge.py

```diff
@@ -0,0 +1,105 @@
+## @ingroup Analyses-Mission-Vary_Cruise

```

Comment:
I wonder if "Vary_Cruise" is the best name, seems short and kind of vague

Reply:
This was a name that Trent picked a long time ago. Maybe Variable_Range_Cruise?

Reply:
Yes, I like Variable_Range_Cruise. I'll approve but make this change